PyDSS/dssElement.py

```diff
@@ -153,11 +153,16 @@ def GetParameter(self, Param):
         if self._dssInstance.Element.Name() != self._FullName:
             self._dssInstance.Circuit.SetActiveElement(self._FullName)
         if self._dssInstance.Element.Name() == self._FullName:
+            # This always returns a string.
+            # The real value could be a number, a list of numbers, or a string.
             x = self._dssInstance.Properties.Value(Param)
             try:
                 return float(x)
-            except:
-                return x
+            except ValueError:
+                try:
+                    return ast.literal_eval(x)
```

Comment:
@AadilLatif This is probably the only change in this PR that is important. This is to fix the transformer-taps bug I mentioned. I basically moved the call to `ast.literal_eval` higher up the stack. It's possible someone could be expecting a string here. I fixed the places that I could find and test.